addons/l10n_cn_small_business/data/account_tax_template_data.xml

```diff
@@ -1,154 +1,65 @@
-    </record>
-    <record id="l10n_cn_small_business_sales_excluded_3" model="account.tax.template">
-        <field name="chart_template_id" ref="l10n_chart_china_small_business"/>
-        <field name="name">税收3％ - 中国小企业会计科目表</field>
-        <field name="description">税收3％</field>
-        <field name="amount">3</field>
-        <field name="amount_type">percent</field>
-        <field name="refund_account_id" ref="l10n_cn_2221"/>
-        <field name="account_id" ref="l10n_cn_2221"/>
-        <field name="type_tax_use">sale</field>
-        <field name="price_include" eval="0"/>
-        <field name="tax_group_id" ref="l10n_cn.l10n_cn_tax_group_vat_3"/>
-    </record>
-
 
-    <!-- purchase tax excluded -->
-    <record id="l10n_cn_small_business_purchase_excluded_17" model="account.tax.template">
+    <record id="vatp_small_business_16" model="account.tax.template">
         <field name="chart_template_id" ref="l10n_chart_china_small_business"/>
-        <field name="name">税收17％ - 中国小企业会计科目表</field>
-        <field name="description">税收17％</field>
-        <field name="amount">17</field>
+        <field name="name">16%进项税 - 中国小企业会计科目表</field>
+        <field name="description">16%进项税</field>
+        <field eval="16" name="amount"/>
         <field name="amount_type">percent</field>
-        <field name="refund_account_id" ref="l10n_cn_2221"/>
-        <field name="account_id" ref="l10n_cn_2221"/>
+        <field name="refund_account_id" ref="small_business2017_chart2221"/>
+        <field name="account_id" ref="small_business2017_chart2221"/>
         <field name="type_tax_use">purchase</field>
-        <field name="price_include" eval="0"/>
-        <field name="tax_group_id" ref="l10n_cn.l10n_cn_tax_group_vat_17"/>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag2')])]"/>
     </record>
-    <record id="l10n_cn_small_business_purchase_excluded_11" model="account.tax.template">
-        <field name="chart_template_id" ref="l10n_chart_china_small_business"/>
-        <field name="name">税收11％ - 中国小企业会计科目表</field>
-        <field name="description">税收11％</field>
-        <field name="amount">11</field>
-        <field name="amount_type">percent</field>
-        <field name="refund_account_id" ref="l10n_cn_2221"/>
-        <field name="account_id" ref="l10n_cn_2221"/>
-        <field name="type_tax_use">purchase</field>
-        <field name="price_include" eval="0"/>
-        <field name="tax_group_id" ref="l10n_cn.l10n_cn_tax_group_vat_11"/>
-    </record>
-    <record id="l10n_cn_small_business_purchase_excluded_6" model="account.tax.template">
+
+    <record id="vats_small_business_13" model="account.tax.template">
         <field name="chart_template_id" ref="l10n_chart_china_small_business"/>
-        <field name="name">税收6％ - 中国小企业会计科目表</field>
-        <field name="description">税收6％</field>
-        <field name="amount">6</field>
+        <field name="name">13％销项税 - 中国小企业会计科目表</field>
```

Comment:
switch name and description,  name should shorter than description.
the same for all  tax.